os/arch/arm/src/artik053/src/artik053_tash.c

```diff
@@ -272,6 +272,7 @@ int board_app_initialize(void)
 	struct mtd_dev_s *mtd;
 #endif /* CONFIG_RAMMTD */
 
+#ifndef CONFIG_DISABLE_MOUNTPOINT
```

Comment:
@sunghan-chang Hello. It seems that you are right and dependency from Kconfig solves the problem. Hence, I'll abandon this change and, as you suggested, will add similar dependency for FLASH_PART into the s5jt200 Kconfig.